pom.xml

```diff
@@ -0,0 +1,703 @@
+    </license>
+  </licenses>
+
+  <!-- Aggregate Modules -->
+  <modules>
+    <module>common</module>
+    <module>credentials</module>
+    <module>crypto</module>
+    <module>dids</module>
+    <module>distribution</module>
+    <module>jose</module>
+    <module>testing</module>
+  </modules>
+
+  <!-- Properties -->
+  <properties>
+    <!-- Project properties -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <kotlin.jvm.target>11</kotlin.jvm.target>
+    <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
+    <!-- Defines the server config in .maven_settings.xml to use in maven-release-plugin -->
+    <project.scm.id>github</project.scm.id>
+
+    <!--
+    This section is where we declare the versioning and scope for dependencies of
+    the Web5 platform and projects building atop the Web5 platform.
+
+    Submodules of Web5 should not define their own dependency versions
+    because these must all co-exist in the same ClassLoading environment, and
+    therefore have to be aligned across submodules. Thus we declare the versioning
+    requirements here at the platform level.
+
+    The <dependencyManagement> section of this POM enforces the versions declared
+    here as properties. Because this POM is a BOM POM, these versions may be imported
+    by projects building atop the Web5 Platform (ie. tbDEX, consumers, etc).
+
+    If a submodule needs to introduce a new dependency or upgrade, define that
+    dependency and version here such that other submodules in the build may pick
+    up the same version. This will guarantee that submodule test suites are running
+    in the correct ClassLoading environment aligned with the Web5 platform.
+
+    More on BOM POMs:
+    https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms
+    -->
+
+    <!-- Versioning for Platform -->
+    <version.kotlin>1.9.22</version.kotlin>
+    <version.io.github.martinvisser>0.0.6</version.io.github.martinvisser>
+
+    <!-- Versioning for Dependencies -->
+    <version.com.amazonaws>1.12.668</version.com.amazonaws>
+    <version.com.danubetech>1.6.0</version.com.danubetech>
+    <version.com.fasterxml.jackson>2.17.0</version.com.fasterxml.jackson>
+    <version.com.github.multiformats>1.1.0</version.com.github.multiformats>
+    <version.com.google.crypto.tink>1.10.0</version.com.google.crypto.tink>
+    <version.com.nimbusds>9.37.2</version.com.nimbusds>
+    <version.com.networknt>1.0.87</version.com.networknt>
+    <version.com.nfeld.jsonpathkt>2.0.1</version.com.nfeld.jsonpathkt>
+    <version.com.squareup.okhttp3>4.12.0</version.com.squareup.okhttp3>
+    <version.dnsjava>3.5.2</version.dnsjava>
+    <version.io.github.erdtman>1.1</version.io.github.erdtman>
+    <version.io.github.oshai>6.0.2</version.io.github.oshai>
+    <version.io.ktor>2.3.7</version.io.ktor>
+    <version.org.assertj>3.25.2</version.org.assertj>
```

Comment:
`assertj` is a test dependency, no?

Reply:
OK, great catch let's move `org.assertj` to the test deps section